tailwind.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
       fontFamily: {
         ar: ['Noto'],
         'ar-title': ['NotoKufi'],
+        am: ['NotoEthio'],
```

Comment:
@sentayhuJo this is `postcss` way to define new classes, it will give us a class called `font-am` that we can use around the app